lib/util/pane-item.js

```diff
@@ -179,14 +179,23 @@ export default class PaneItem {
     return this.__title
   }
 
+  getDefaultLocation() {
+    return this.__defaultLocation
+  }
+
+  setDefaultLocation(defaultLocation) {
+    this.__defaultLocation = defaultLocation
+    this.__emitter.emit('change-default-location', defaultLocation)
```

Comment:
Just for consistency: This event should be called `didChangeDefaultLocation`.

Reply:
Okay. So `emit('did-change-default-location')` would be fine ??
(Sorry but I still don't track well about what this emitter exactly does...)

Reply:
I actually don't think you need to emit an event here at all, unless you want to consume it in julia-client.
And no, I literally meant `this.__emitter.emit('didChangeDefaultLocation', defaultLocation)` (although the `change-title` thing is inconsistent with that too, so don't worry about it :D).

Reply:
Aah, now I understand.
And haha, okay. I remove this emission.
Thanks for pointing this out !

Reply:
Btw, there's docs for the pane item API [here](https://github.com/atom/atom/blob/v1.37.0/src/workspace.js#L22-L174). So we have the emitter so that Atom can subscribe to title change events etc.